.github/workflows/js-ci.yml

```
@@ -282,7 +282,7 @@ jobs:
           wait-on: http://localhost:8080
           working-directory: js/apps/admin-ui
         env:
-          CYPRESS_BASE_URL: http://localhost:8080/admin/
+          #CYPRESS_BASE_URL: http://localhost:8080/admin/
```

Probably committed by accident?
```suggestion
          CYPRESS_BASE_URL: http://localhost:8080/admin/
```

---

No, this is intentional. Without it, the https://github.com/keycloak/keycloak/actions/runs/9710664453?pr=30893 shows that the `/admin` is included twice.